corehq/apps/data_interfaces/interfaces.py

```diff
@@ -30,6 +31,15 @@ def default_report_url(self):
         return reverse('data_interfaces_default', args=[self.request.project])
 
 
+class ArchiveFormInterface(DataInterface):
+    name = ugettext_noop("Bulk Archive Forms")
+    slug = "archive_forms"
+
+    @classmethod
+    def show_in_navigation(cls, domain=None, project=None, user=None):
+        return BULK_ARCHIVE_FORMS.enabled(user.username)
```

Comment:
one more thing - you should also make sure the toggle and subscription check are honored even if you navigate to the right url manually (so need to do an additional check in the view)